server/src/models/configModel.ts

```diff
@@ -1,17 +1,26 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export type PostCallSurveyType = 'msforms' | 'custom';
-interface MSFormsSurveyOptions {
+export type PostCallSurveyType = 'msforms' | 'custom' | 'onequestionpoll';
+export interface MSFormsSurveyOptions {
   surveyUrl: string;
 }
-interface CustomSurveyOptions {
+export interface CustomSurveyOptions {
   surveyUrl: string;
 }
+
+export type OneQuestionPollType = 'likeorDislike' | 'rating' | 'text';
```

Comment:
Should we capitalize the 'o' in `likeOrDislike`?

Reply:
Fixed this.